Extensions/Today/ImageButtonWithLabel.swift

```diff
@@ -3,7 +3,6 @@
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
-
```

Comment:
nit: This line was fine to keep, we do add one space its the line after class that we don't need space on.